app/uk/gov/hmrc/plasticpackagingtax/returns/views/returns/converted_packaging_credit_page.scala.html

```diff
@@ -75,6 +78,18 @@
     )
 
 }
+@linkHint = {<a href=@messages("returns.convertedPackagingCredit.details.href") class="govuk-link" id="converted-plastic-weight-readMore-link">@messages("returns.convertedPackagingCredit.details.readMore")</a>}
+
+@formattedDetailsContent = {
+    @paragraph(content = Html(messages("returns.convertedPackagingCredit.details.line1")))
+
+    <ul class="govuk-list govuk-list--bullet">
+        <li>@messages("returns.convertedPackagingCredit.details.bullet1")</li>
+        <li>@messages("returns.convertedPackagingCredit.details.bullet2")</li>
+    </ul>
+
+    @paragraph(linkHint)
+}
 
 @govukLayout(
     title = Title("returns.convertedPackagingCredit.meta.title"),
```

Comment:
This missed the "Submit return key". The Title components take few parameter. I added this for another page as following:

title = Title(
                "returns.humanMedicinesPlasticWeight.meta.title",
                sectionKey = "service.section")

I have added the key service.section on PPTP-1779 branch